WcaOnRails/spec/support/database_cleaner.rb

```diff
@@ -1,22 +1,23 @@
 RSpec.configure do |config|
+  reference_tables_to_keep = %w(Countries Continents Events Rounds Formats teams)
   config.before(:suite) do
-    DatabaseCleaner.clean_with(:truncation)
+    DatabaseCleaner.clean_with(:truncation, except: reference_tables_to_keep)
+    reference_tables_to_keep.each do |table|
+      ActiveRecord::Base.connection.execute("TRUNCATE #{table};")
+      load "#{Rails.root}/db/seeds/#{table.underscore}.seeds.rb"
```

Comment:
@larspetrus could you please let me know a bit more about how this should work? Does that mean that we'd need to ensure that the seeds files and the PopulateTestDb.rb would need to kept in sync?

We also don't have a method for creating Continents, Countries, Events, Rounds, and Formats so I guess we'd need to use `ActiveRecord::Base.connection.execute`?